torchci/lib/bot/acceptBot.ts

```diff
@@ -7,7 +7,7 @@ function containsLabel(labels: Label[], labelName: string) {
 
 export const ACCEPT_2_RUN = "accept2run";
 export const ACCEPT_2_SHIP = "accept2ship";
-export const CIFLOW_ALL = "ciflow/all";
+export const CIFLOW_LABEL = "ciflow/trunk";
```

Comment:
```suggestion
export const CIFLOW_TRUNK_LABEL = "ciflow/trunk";
```

since there's also ciflow/scheduled etc.